examples/speed_tester/speed_tester.c

```diff
@@ -472,10 +483,11 @@ nf_setup(struct onvm_nf_info *nf_info) {
                         }
                         pkts[i] = pkt;
                 }
-                onvm_nflib_return_pkt_bulk(nf_info, pkts, packet_number);
+                onvm_nflib_return_pkt_bulk(nf_info, pkts, pkts_generated);
```

Comment:
lets move the `onvm_nflib_return_pkt_bulk` function out of the ifdef statement as its the same for both

Reply:
When I do this, it seems to think that pkts is declared but never used, any suggestions? The other fixes seem to be fine

Reply:
If its being weird because of our #ifdef structure that its okay, we can keep it

Reply:
I think we can just take out the #ifdef LIBPCAP and keep the `pcap_filename != NULL`, then in the initialization of pkts, test whether the packet_number is changed? I can test and push to see your thoughts

Reply:
The #ifdef LIBPCAP allows us to build without a pcap library

Reply:
Oh ok that makes sense